pkgs/applications/audio/easyabc/default.nix

```diff
@@ -1,7 +1,22 @@
-{ lib, fetchFromGitHub, substituteAll, makeWrapper, python39, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
+{ lib, fetchFromGitHub, fetchPypi, substituteAll, makeWrapper, python39, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
 
-# requires python39 due to https://stackoverflow.com/a/71902541 https://github.com/jwdj/EasyABC/issues/52
-python39.pkgs.buildPythonApplication {
+let
+  # requires python39 due to https://stackoverflow.com/a/71902541 https://github.com/jwdj/EasyABC/issues/52
+  python = python39.override {
+    packageOverrides = self: super: {
+      # currently broken with 4.2.1
+      # https://github.com/jwdj/EasyABC/issues/75
+      wxPython = self.wxPython_4_2.overrideAttrs (args: rec {
```

Comment:
```suggestion
      wxPython_4_2 = self.wxPython_4_2.overrideAttrs (args: rec {
```

Reply:
If I apply this and the other changes I get an infinite recursion :man_shrugging:

Reply:
`wxPython_4_2 = super.wxPython_4_2.overrideAttrs` 

note: `super`

Reply:
applied in https://github.com/NixOS/nixpkgs/pull/243787/commits/6857e5dff352e1e5fda1b6e2fc33b69ac3b38bd9